src/components/AttentionBox/AttentionBox.jsx

```diff
@@ -44,23 +48,36 @@ const AttentionBox = ({
   return (
     <aside
       className={cx(
+        styles.box,
```

Comment:
let's change the name to attentionBox because box is a different component and it can be confusing in debug and investigate issues